core/src/main/java/org/openstack4j/openstack/networking/domain/ext/NeutronLbPoolV2.java

```diff
@@ -148,6 +154,16 @@ public String getHealthMonitorId(){
     }
 
     @Override
+    public String getSubnetId() {
```

Comment:
If subnetId and vipId are not actually in the response, then I can revert those. Let me know after you do some testing.